modules/cluster-logging-curator-tolerations.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-curator.adoc
+
+[id="cluster-logging-curator-tolerations_{context}"]
+= Using tolerations to control the Curator pod placement
+
+You can control which node the Curator pod runs on
+using tolerations on the pod.
+
+You apply tolerations to the Curator pod through the Cluster Logging Custom Resource (CR)
+and apply taints to a node through the node specification. A taint on a node 
+instructs the node to repel all pods that do not tolerate the taint.
+
+
+.Prerequisites
+
+* Cluster logging and Elasticsearch must be installed.
+
+.Procedure
+
+. Use the following command to add a taint to a node where you want the Curator pod scheduled:
++
+----
+$ oc adm taint nodes <node-name> <key>=<value>:<effect>
+----
++
+For example:
++
+----
+$ oc adm taint nodes node1 key1=value1:NoExecute
+----
++
+This example places a taint on `node1` that has key `key1`, value `value1`, and taint effect `NoExecute`.
+Nodes with the `NoExecute` effect schedule only pods that match the taint and remove existing pods
+that do not match.
+
+. Edit the `curation` section of the Cluster Logging Custom Resource (CR) to configure a toleration for the Curator pod:
++
+[source, yaml]
+----
+  curation:
+    type: "curator"
+    curator:
+      tolerations: 
+      - key: "logging"  <1>
+        operator: "Exists"  <2>
+        effect: "NoExecute"  <3>
+        tolerationSeconds: 6000  <4>
+----
+<1> Specify the key you added to the node.
+<2> Specify the `Exists` operator to require the `key`/`value`/`effect` parameters to match. 
+<3> Specify the `NoExecute` effect.
+<4> Optionally, specify the `tolerationSeconds` parameter to set how long a pod can remain bound to a node before being evicted.
+
+This toleration matches the taint created by the `oc adm taint` command above. A pod this toleration would be able to schedule onto `node1`.
```

Comment:
s/created/that is created

Not "above"
> ! above adj
Do not use to indicate a relative location in a document, as in “the above restrictions.” Use “previous” or
“preceding.”